src/dsp_tools/commands/xmlupload/stash/upload_stashed_xml_texts.py

```diff
@@ -86,7 +86,7 @@ def _create_XMLResource_json_object_to_update(
         link_prop_name: {
             "@id": value_iri,
             "@type": "knora-api:TextValue",
-            "knora-api:textValueAsXml": '<?xml version="1.0" encoding="UTF-8"?>\n<text>' + str(new_xmltext) + "</text>",
+            "knora-api:textValueAsXml": new_xmltext.as_xml(),
```

Comment:
@Nora-Olivia-Ammann : Could you update the PR title so that it is understandable for people who are not core devs of dsp-tools? And the same for the Linear ticket. Thanks :-)